CMakeLists.txt

```diff
@@ -139,6 +140,7 @@ endif()
 if(CORE_SYSTEM_NAME STREQUAL windowsstore)
   #uses header only version of Fmt
   list(REMOVE_ITEM required_deps Fmt)
+  list(REMOVE_ITEM CORE_REQUIRED_DEPS fmt)
```

Comment:
no this is the wrong fix, correct is removing this hole hack

Reply:
I agree, but I have no idea why uwp uses this and I'm certainly not going to touch windows deps.

Reply:
then wait with merging, I'll PR the removal

Reply:
great, thanks